x/account/keeper/msg_server_create_coordinator.go

```diff
@@ -0,0 +1,47 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tendermint/spn/x/account/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func (k msgServer) CreateCoordinator(
+	goCtx context.Context,
+	msg *types.MsgCreateCoordinator,
+) (*types.MsgCreateCoordinatorResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	// Validate the coordinator address
+	_, err := sdk.AccAddressFromBech32(msg.Address)
```

Comment:
We don't need this check since it's already performed here: https://github.com/tendermint/spn/blob/b87cc435681cfe48f2d8ed8f5936ddcd54c3c07c/x/account/types/message_create_coordinator.go#L42

More generally we should keep all stateless checks under `types/` and checks that depend on the state of the app under `keeper/`